src/common/libutil/sigutil.c

```diff
@@ -0,0 +1,127 @@
+/************************************************************\
+ * Copyright 2023 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <errno.h>
+#include <signal.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "ccan/str/str.h"
+#include "ccan/array_size/array_size.h"
+
```

Comment:
sigutil.h should be included to ensure prototypes match definitions.

Reply:
Oh, good catch.